Assets/Scripts/DamColl.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SocialPlatforms.Impl;
+using UnityEngine.UI;
+
+public class DamColl : MonoBehaviour
+{
+    // varibles for enemy ship
+    int enemyShip = 1;
+    int enemyPoints;
+    public Text  Score;
+
+    private void Start()
+    {
+        Score.text = "Score";
+    }
+
+
+
+    private void OnTriggerEnter2D(Collider2D collision)
```

Comment:
I understand that this is on each enemy game object so that you can control what happens when it gets collided with.  But the way this is written it will interact with any collision event .  In the future you should try to use the collision access the game object that was collided with and then determine what to do with it. For example if you had a projectile tag on what ever is interacting within the collider here you could say

```
private void OnTriggerEnter2D(Collider2D collision)
{
   if(collision.gameObject.CompareTag("projectile")
   {
        enemyShip--;
    }
}

```